### ISSUE TITLE
seems the partition column's value miss when row.get method

### ISSUE THREAD

**Issue by wwang-talend** [delta-io/connectors#126]
```
Snapshot snapshot = log.snapshot();
...
RowRecord row = iter.next();
row.getString("any_partition_column")
//here no any way to fetch partioner column's value, just return null
```
And the schema/metadata api contains that column

any advice?

**Comment by wwang-talend** [comment]
this is the pr to support that : https://github.com/delta-io/connectors/pull/127 ,  please help review

**Comment by zsxwing** [comment]
Fixed by #197
